### PR TITLE
ergoCubEmotions module - stint 2

### PR DESCRIPTION
**Pull request by martinaxgloria**
The module is now refactored and the images are stored in a context called `ergoCubEmotions` and installed by means of `yarp_install`. In this way, ResourceFinder finds and reads the configuration file which is installed in a YARP's installation dir.

### CODE REVIEW

File: src/modules/ergoCubEmotions/ergoCubEmotions.xml
Comment (pattacini) [inline src/modules/ergoCubEmotions/ergoCubEmotions.xml:1]: This documentation needs to be adapted to the new module 😉

File: src/modules/ergoCubEmotions/idl.thrift
Comment (pattacini) [inline src/modules/ergoCubEmotions/idl.thrift:1]: You may consider documenting the API using the javadoc, which si recognized by doxygen.
Get inspired for this from the original code.

**Review by pattacini — CHANGES_REQUESTED** [review]
Fantastic job @martinaxgloria 🚀 
I would only refine the documentation. Please, follow the hints a left inline.

File: src/modules/ergoCubEmotions/ergoCubEmotions.xml
Comment (pattacini) [reply src/modules/ergoCubEmotions/ergoCubEmotions.xml:1]: Read carefully the content of the file for this.

File: src/modules/ergoCubEmotions/idl.thrift
Comment (martinaxgloria) [reply src/modules/ergoCubEmotions/idl.thrift:1]: Then simply by hand. I thought that it was an automatic way to generate those comments.
Ok, thanks @pattacini!

Comment (pattacini) [reply src/modules/ergoCubEmotions/idl.thrift:1]: The other way around: those comments will be used at a second stage by doxygen to generate the online documentation. For the latter step, hold on; just focus on inserting the javadoc comments inline.

**Review by pattacini — CHANGES_REQUESTED** [review]
Just a minor fix.

**Review by pattacini — APPROVED** [review]

**Review by Nicogene — APPROVED** [review]

**Comment by Nicogene** [comment]
Merged, thanks!

**Pull request merged**